pallets/gear-bank/src/tests.rs

```diff
@@ -554,6 +554,8 @@ fn spend_gas_small_amount_validator_account_deleted() {
 #[test]
 fn spend_gas_zero() {
     new_test_ext().execute_with(|| {
+        let _block_author = Authorship::author();
```

Comment:
nope, it was added to not break the tests since getting block author changes storage state but the tests check there were no modifications of the storage.

Frankly speaking it is a consequence of `spend_gas` refactoring